recipes/dbus/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(DBus1 REQUIRED)
+
+include_directories(${DBus1_INCLUDE_DIRS})
+
+add_executable(example example.cpp)
+target_link_libraries(example ${DBus1_LIBRARIES})
```

Comment:
Doesn't/Shouldn't Dbus1 create a `dbus1::dbus1` target (or something similar)?
That would avoid the `include_directories` call, and you can link directly to `dbus1::dbus1`.

Reply:
I didn't find anything about dbus1::dbus1, so I've just added this example from stackoverflow (https://stackoverflow.com/questions/38515316/how-to-configure-dbus-dependencies-with-cmake)

Reply:
I just checked the dbus1 of Fedora Linux (my distro).
See https://fedora.pkgs.org/31/fedora-x86_64/dbus-devel-1.12.16-3.fc31.x86_64.rpm.html
It distributes `DBus1Config.cmake` and `dbus-1.pc`, so the cmake name should be `DBus1`, the pkg_config name `dbus-1`.
Inside the cmake script, it only creates a `dbus-1` library.

Reply:
So you should be able to do:
```
find_package(DBus1 REQUIRED)

add_executable(example example.cpp)
target_link_libraries(example dbus-1)
```